docs/atoz.md

```diff
@@ -0,0 +1,487 @@
+
+Copy the key to our `~/.ssh/authorized_keys` file, using this command:
+
+~~~bash
+cat ~/.ssh/wetty.pub >> ~/.ssh/authorized_keys
+~~~
+
+Now give these file and folders the correct permissions, using these commands:
+
+~~~bash
+chmod 700 ~/.ssh
+chmod 644 ~/.ssh/authorized_keys
+chmod 600 ~/.ssh/wetty
+~~~
+
+**Optional:** A housekeeping command. If you need to remove all entries of the `wetty` public key with the comment `wetty-keyfile` from the `~/.ssh/authorized_keys` file use this command. Otherwise ignore this.
+
+~~~bash
+sed -r '/^ssh-ed25519(.*)wetty-keyfile$/d' -i ~/.ssh/authorized_keys
+~~~
+
+### Install wetty
+
+**Note:** we are using `-g` for `npm` or `global` for `yarn` along with `--prefix ~/` so that the application symbolic link is installed to our `~/bin` directory and available in our local user's `PATH`.
+
+As your local user run these commands to install `wetty` and `forever`. We will need `forever` later to run wetty in the background.
+
+First, we need to make sure the local user's `~/bin` folder exists and is in the `PATH` for the following commands to work.
+
+~~~bash
+mkdir -p ~/bin && source ~/.profile
+~~~
+
+Please use either the `npm` or `yarn` method and not both. The `yarn` method is recommended but I provide both as you may have a personal preference. The outcome is effectively the same.
+
+`npm `  - optional - use `npm` to install wetty
+
+~~~bash
+npm install -g wetty forever --prefix ~/
+~~~
+
+`yarn` -  recommended - use `yarn` to install wetty
+
+~~~bash
+npm install -g yarn --prefix ~/
+yarn global add wetty forever --prefix ~/
+~~~
+
+Once successfully installed the application should be available in your local user's `PATH`. To test the installation was successful please use this command:
+
+~~~bash
+wetty -h
+~~~
+
+### Accessing the web interface.
+
+This needs to be done here because it is not easy to do in the next steps if `wetty` is running in the terminal.
+
+This command will generate the correct URL you need to visit after using the start up commands in the following section.
+
+~~~bash
+echo https://$(curl -s4 icanhazip.com):3000
+~~~
+
+*Please make make a note of this URL now.*
+
+### Running wetty
+
+Now we have all the ground work done we can focus on our `wetty` server configuration settings.
+
+For example, the below command would provide a `https` instance with automatic `ssh` authorisation using our `wetty` private key on port `3000` accessible at `https://IP:3000` . Refer to the previous URL generation command.
+
+**Important note:** This command will run in your current terminal session and not in the background.
+
+~~~bash
+wetty --host 0.0.0.0 -p 3000 --title wetty --base / --sshkey ~/.ssh/wetty --sshhost localhost --sshuser $(whoami) --sshport 22 --sshauth publickey --sslkey ~/.ssl/wetty.key --sslcert ~/.ssl/wetty.crt
+~~~
+
+#### forever to manage wetty
```

Comment:
I normally recommend using systemd  which you should be able to use a systemd user service